### ISSUE TITLE
FrameSwitcher Remoting Diagnostic Action - what is the use case?

### ISSUE THREAD

**Issue by muescha** [krasa/FrameSwitcher#41]
can you explain for what the use case and when I can use this action:

- `FrameSwitcher Remoting Diagnostic Action`

**Comment by krasa** [comment]
No use case for users. It just prints org.jgroups.JChannel properties.

**Comment by muescha** [comment]
For what is the JChannel and the remote / networking stuff?

Is it for `Integrate multiple IDE instances`?

How does this function work? There should be a short info or some instructions in the plugin description how to use it.

**Comment by krasa** [comment]
yes, you turn it on, and it should work, but it is a little bugged 

![obrazek](https://github.com/krasa/FrameSwitcher/assets/1160875/048118da-41d3-4379-b265-f4161e71834d)
